docs/coordinates/solarsystem.rst

```diff
@@ -5,47 +5,91 @@
 Solar System Ephemerides
 ------------------------
 
-`astropy.coordinates` can calculate the |SkyCoord| of some of the major
-solar system objects. Coordinates are calculated using the JPL DE430 
-ephemerides. These ephemerides provide predictions valid roughly for years 
-between 1550 and 2650. The file is 115 MB and will need to be downloaded 
-the first time you use this functionality, but will be cached after that.
+`astropy.coordinates` can calculate the |SkyCoord| of some of the major solar
+system objects. By default, it uses approximate orbital elements from `ERFA
+<https://github.com/liberfa/erfa>`_ routines, but one can also use more precise
```

Comment:
"one can" -> "it can" (for consistency with the previous sentence, where `astropy.coordinates` is the "actor".